src-backend/api/signals/page_pre_save.py

```diff
@@ -0,0 +1,16 @@
+from django.dispatch import receiver
+from django.db.models.signals import pre_save
+from django.db.models import F
+from api.models import Page
+import logging
+
+logger = logging.getLogger('debugger')
+
+
+@receiver(pre_save, sender=Page)
+def on_page_pre_save(sender, instance=None, raw=False, **kwargs):
+    # Update index of all objects with display index >= object we are saving
+    if instance.display_index is not None:
```

Comment:
Does this mean if we try to save a page with a `display_index` it'll do the updates?